### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN error

### DISCUSSION THREAD

**Discussion by kkw0432** [vercel/community#6807] [Bug Report]
### Page to Investigate

https://queens-web.vercel.app

### Steps to Reproduce

my vercel app has DNS problem.
it work well yesterday but when I tried to access it now, it failed.
our app already deployed using that domain, so I can not switch to other domain now..
Please check and resolve this issue quickly. 

Thank you

**Comment by spoyodevelop** [comment]
Seems like and somehow our korean ISPs blocked vercels DNS(vercel.app) for some dumb reason. so basically you can use any https bypass app like icloud Private Relay or unicorn https to connect in your website. 
which is not a solution or anything, but still we can know where exactly problem is.
![스크린샷 2024-05-11 11 37 32](https://github.com/vercel/community/assets/73924592/570cd455-333e-42c6-9603-90f1efc50c61)

**Reply by spoyodevelop** [reply]
Probably just DNS being weird i believe, i have no clue what is going on, just website are unreachable in only south korea

**Reply by spoyodevelop** [reply]
Use https://www.digitalocean.com/ or https://netlify.com/ for alternative. 
Here is link for who are interested in digitalocean deployment https://youtube.com/watch?v=1ldjq5M7cbw

**Reply by spoyodevelop** [reply]
or use custom domain whichever not using (vercel.app) if you can.

**Reply by kkw0432** [reply]
oh... is there no way but to wait?
as I metioned my app is using this domain..
I needs app update to change the domain url.. it will take few days. in addition it is weekend.
if it takes a lot of time, I'll prepare an app update

**Reply by kkw0432** [reply]
I'm sorry service has been stopped so long time..

**Reply by kkw0432** [reply]
now it is working well.
I will change to custom domain so that I can deal with like this status.

**Comment by SeokyoungYou** [comment]
I am in Korea and also could not access the preview URL using the Vercel domain.

**Comment by safethecode** [comment]
The project with the connected domain seems fine, but accessing it via *.vercel.app address doesn't seem to work.

**Comment by idchoi2** [comment]
fuck korean ISPs

**Comment by PurpleSand123** [comment]
Me too. My [vercel app](https://bequiet-log.vercel.app/) has same issue

**Comment by makepado** [comment]
Korea did Korea

**Comment by Nangniya** [comment]
Me too. I purchased a different domain name and connected it newly. 😢

**Reply by Socrad** [reply]
Me too. I had to change nginx config of backend server for CORS.
Now I'm afraid of missing something

**Comment by hiddenest** [comment]
For the people who are experiencing this issue:
Could you run this bash commands on your computer and share the result here?
It would be really helpful for Vercel team to resolve this issue.

https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1

**Comment by hiddenest** [comment]
@spoyodevelop It looks like you are using U+(LG Dacom) ISP.
In the last 2 week, KT/SKT/SKB had blocked `76.76.21.22` domain, which is locating to one of Vercel CNAME IP address. (Now it is resolved) [(Issue Link)](https://x.com/KrComet/status/1786931094178410695)

My guess is that the Korean government asked ISPs to block that IP addresses.
...and it's a problem now because U+ reflected it slowly compared to KT/SKT.

**Reply by Socrad** [reply]
No. My vercel domain also has same problem from today morning, and everyone in Seoul(maybe whole Korea) can't connect to the service. I'm using SKT, and the other using KT, and everyone has same problem.

**Reply by spoyodevelop** [reply]
I am used that website for better visual representation, not for very accurate information so take them with grain of salt. I am currently using KT and I tried other ISPs and seems no one works.

**Reply by hiddenest** [reply]
Hmm... It could be quite difficult to debug it then.
Because when I made this comment, I've tested it on both KT and SKT and there was no problem (it is working now, too)

**Comment by sko-kr** [comment]
Seriously, I feel like we have dictatorship when it comes to ISPs in Korea.
They blocked r2storage (cloudflare r2 storage) now vercel.app.

**Comment by henrynoowah** [comment]
Hi, guys, I just checked and want to let y'all know that it is now working

**Reply by kdh8219** [reply]
It still not working on my machine.
isp:kt

**Reply by Socrad** [reply]
I checked, yes, *.vercel.app domain is working now. But still don't know why. Is there anyone who knows why this accident happened?

**Reply by spoyodevelop** [reply]
It do not works on my machine, i am using KT as ISP now too.

**Reply by spoyodevelop** [reply]
No idea but this might give you hint
https://www.reddit.com/r/nextjs/comments/pl11tc/vercels_ip_is_blacklisted_and_inaccessible_from/

**Reply by Socrad** [reply]
dnschecker.org  still shows the domains don't working in Korea.

**Reply by Socrad** [reply]
And other people says it doesn't working.

**Reply by kkw0432** [reply]
my project is working now.

**Comment by mcsdevv** [comment]
Hi all, sorry to hear you're once again having difficulty - we're more than happy to take this to the ISP once we have a little more detail.

As @hiddenest mentioned, can those unable to access `vercel.app` domains run the following commands? https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1

This will allow us to take more evidence to the ISP that there is indeed an issue and aim to resolve it quicker.

**Reply by spoyodevelop** [reply]
```
Last login: Sat May 11 08:33:22 on console
Spoy@Marvins-MacBookAir ~$ date
2024년 5월 11일 토요일 21시 44분 37초 KST
Spoy@Marvins-MacBookAir ~$ curl https://ipinfo.io
{
  "ip": "thisismyIP"
  "city": "Yongsan-dong",
  "region": "Seoul",
  "country": "KR",
  "loc": "37.4988,127.1401",
  "org": "AS4766 Korea Telecom",
  "postal": "05742",
  "timezone": "Asia/Seoul",
  "readme": "https://ipinfo.io/missingauth"
}%
Spoy@Marvins-MacBookAir ~$ ping -c 4 76.76.21.21
PING 76.76.21.21 (76.76.21.21): 56 data bytes
64 bytes from 76.76.21.21: icmp_seq=0 ttl=241 time=34.782 ms
64 bytes from 76.76.21.21: icmp_seq=1 ttl=241 time=34.755 ms
64 bytes from 76.76.21.21: icmp_seq=2 ttl=241 time=35.640 ms
64 bytes from 76.76.21.21: icmp_seq=3 ttl=241 time=36.029 ms

--- 76.76.21.21 ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 34.755/35.302/36.029/0.551 ms
Spoy@Marvins-MacBookAir ~$ ping -c 4 cname.vercel-dns.com
PING cname.vercel-dns.com (76.76.21.98): 56 data bytes
64 bytes from 76.76.21.98: icmp_seq=0 ttl=241 time=35.366 ms
64 bytes from 76.76.21.98: icmp_seq=1 ttl=241 time=35.303 ms
64 bytes from 76.76.21.98: icmp_seq=2 ttl=241 time=35.614 ms
64 bytes from 76.76.21.98: icmp_seq=3 ttl=241 time=35.288 ms

--- cname.vercel-dns.com ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 35.288/35.393/35.614/0.131 ms
Spoy@Marvins-MacBookAir ~$ dig didibringmyumbrella.vercel.app/geocode

; <<>> DiG 9.10.6 <<>> didibringmyumbrella.vercel.app/geocode
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 24551
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;didibringmyumbrella.vercel.app/geocode.	IN A

;; AUTHORITY SECTION:
.			10800	IN	SOA	a.root-servers.net. nstld.verisign-grs.com. 2024051100 1800 900 604800 86400

;; Query time: 5 msec
;; SERVER: 192.168.50.1#53(192.168.50.1)
;; WHEN: Sat May 11 21:46:15 KST 2024
;; MSG SIZE  rcvd: 142

Spoy@Marvins-MacBookAir ~$ dig didibringmyumbrella.vercel.app/geocode @8.8.8.8

; <<>> DiG 9.10.6 <<>> didibringmyumbrella.vercel.app/geocode @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 58551
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;didibringmyumbrella.vercel.app/geocode.	IN A

;; AUTHORITY SECTION:
.			86399	IN	SOA	a.root-servers.net. nstld.verisign-grs.com. 2024051100 1800 900 604800 86400

;; Query time: 63 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun May 12 01:13:30 KST 2024
;; MSG SIZE  rcvd: 142

Spoy@Marvins-MacBookAir ~$ curl -sv https://didibringmyumbrella.vercel.app
* Could not resolve host: didibringmyumbrella.vercel.app
* Closing connection

Spoy@Marvins-MacBookAir ~$ traceroute 76.76.21.21
traceroute to 76.76.21.21 (76.76.21.21), 64 hops max, 40 byte packets
 1  192.168.219.1 (192.168.219.1)  4.732 ms  2.955 ms  4.175 ms
 2  124.55.173.65 (124.55.173.65)  6.159 ms  6.972 ms *
 3  10.204.232.201 (10.204.232.201)  6.692 ms  5.537 ms  6.141 ms
 4  1.213.10.117 (1.213.10.117)  6.447 ms
    1.208.10.113 (1.208.10.113)  6.908 ms
    1.213.10.117 (1.213.10.117)  6.374 ms
 5  1.213.114.193 (1.213.114.193)  6.690 ms
    1.213.152.249 (1.213.152.249)  6.768 ms
    1.213.105.125 (1.213.105.125)  6.030 ms
 6  * 1.208.179.138 (1.208.179.138)  7.656 ms
    1.208.113.118 (1.208.113.118)  8.636 ms
 7  * 211.60.104.22 (211.60.104.22)  8.648 ms
    211.50.0.198 (211.50.0.198)  6.021 ms
 8  52.93.248.2 (52.93.248.2)  7.208 ms *
    52.93.248.20 (52.93.248.20)  90.014 ms
 9  * * *
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * * *
30  * * *
31  * * *
32  * * *
33  *^C

```

traceroute stops from south korea.

**Reply by winterlood** [reply]
In my situation, I had access starting from 11 p.m. yesterday (in KST)
I'm not sure if this is useful, but I'm sending you the outcomes from executing the specified command.

```
 정환  > date
2024년 5월 12일 일요일 10시 22분 20초 KST


 정환 > curl https://ipinfo.io
{
  "ip": "115.21.67.87",
  "city": "Seoul",
  "region": "Seoul",
  "country": "KR",
  "loc": "37.5660,126.9784",
  "org": "AS4766 Korea Telecom",
  "postal": "03141",
  "timezone": "Asia/Seoul",
  "readme": "https://ipinfo.io/missingauth"
}%


 정환 > ping -c 4 76.76.21.21
PING 76.76.21.21 (76.76.21.21): 56 data bytes
64 bytes from 76.76.21.21: icmp_seq=0 ttl=241 time=29.749 ms
64 bytes from 76.76.21.21: icmp_seq=1 ttl=241 time=30.498 ms
64 bytes from 76.76.21.21: icmp_seq=2 ttl=241 time=30.717 ms
64 bytes from 76.76.21.21: icmp_seq=3 ttl=241 time=30.377 ms

--- 76.76.21.21 ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 29.749/30.335/30.717/0.360 ms


 정환 > ping -c 4 cname.vercel-dns.com
PING cname.vercel-dns.com (76.76.21.22): 56 data bytes
64 bytes from 76.76.21.22: icmp_seq=0 ttl=240 time=26.109 ms
64 bytes from 76.76.21.22: icmp_seq=1 ttl=240 time=26.438 ms
64 bytes from 76.76.21.22: icmp_seq=2 ttl=240 time=25.523 ms
64 bytes from 76.76.21.22: icmp_seq=3 ttl=240 time=25.778 ms

--- cname.vercel-dns.com ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 25.523/25.962/26.438/0.345 ms


 정환 > dig winterlood-com.vercel.app

; <<>> DiG 9.10.6 <<>> winterlood-com.vercel.app
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 18211
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;winterlood-com.vercel.app.	IN	A

;; ANSWER SECTION:
winterlood-com.vercel.app. 600	IN	A	76.76.21.22
winterlood-com.vercel.app. 600	IN	A	76.76.21.61

;; Query time: 35 msec
;; SERVER: 168.126.63.1#53(168.126.63.1)
;; WHEN: Sun May 12 10:24:09 KST 2024
;; MSG SIZE  rcvd: 86


 정환 > dig winterlood-com.vercel.app @8.8.8.8

; <<>> DiG 9.10.6 <<>> winterlood-com.vercel.app @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 44119
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;winterlood-com.vercel.app.	IN	A

;; ANSWER SECTION:
winterlood-com.vercel.app. 600	IN	A	76.76.21.123
winterlood-com.vercel.app. 600	IN	A	76.76.21.164

;; Query time: 38 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun May 12 10:25:05 KST 2024
;; MSG SIZE  rcvd: 86


 정환 > curl -sv https://winterlood-com.vercel.app
*   Trying 76.76.21.93:443...
* Connected to winterlood-com.vercel.app (76.76.21.93) port 443 (#0)
* ALPN: offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=*.vercel.app
*  start date: Apr 15 01:56:22 2024 GMT
*  expire date: Jul 14 01:56:21 2024 GMT
*  subjectAltName: host "winterlood-com.vercel.app" matched cert's "*.vercel.app"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* using HTTP/2
* h2 [:method: GET]
* h2 [:scheme: https]
* h2 [:authority: winterlood-com.vercel.app]
* h2 [:path: /]
* h2 [user-agent: curl/8.1.2]
* h2 [accept: */*]
* Using Stream ID: 1 (easy handle 0x12c012e00)
> GET / HTTP/2
> Host: winterlood-com.vercel.app
> User-Agent: curl/8.1.2
> Accept: */*
>
< HTTP/2 308
< cache-control: public, max-age=0, must-revalidate
< content-type: text/plain
< date: Sun, 12 May 2024 01:25:33 GMT
< location: /post
< refresh: 0;url=/post
< server: Vercel
< strict-transport-security: max-age=63072000; includeSubDomains; preload
< x-vercel-id: kix1::6fvkb-1715477133741-6263aceec9ce
<
Redirecting...
```

**Reply by smaeda-ks** [reply]
@spoyodevelop Your dig command is wrong. It shouldn't be a URL, just the domain name. Could you get that?

```
$ dig didibringmyumbrella.vercel.app

$ dig didibringmyumbrella.vercel.app @8.8.8.8
```

Anyway, seems they put DNS filtering for `*.vercel.app` domains. If switching to a public DNS service (e.g., 8.8.8.8) works then that's most likely it. I'm not seeing it from @winterlood 's latest result, but if you're still seeing issues please get fresh one so we can reach out to the ISP as needed.

**Reply by spoyodevelop** [reply]
```
Spoy@Marvins-MacBookAir ~$ dig vercel.app

; <<>> DiG 9.10.6 <<>> vercel.app
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 49765
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;vercel.app.			IN	A

;; ANSWER SECTION:
vercel.app.		600	IN	A	76.76.21.164
vercel.app.		600	IN	A	76.76.21.9

;; Query time: 34 msec
;; SERVER: 192.168.50.1#53(192.168.50.1)
;; WHEN: Sun May 12 12:23:57 KST 2024
;; MSG SIZE  rcvd: 71

Spoy@Marvins-MacBookAir ~$ dig vercel.app @8.8.8.8

; <<>> DiG 9.10.6 <<>> vercel.app @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 22301
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;vercel.app.			IN	A

;; ANSWER SECTION:
vercel.app.		555	IN	A	76.76.21.93
vercel.app.		555	IN	A	76.76.21.61

;; Query time: 33 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Sun May 12 12:24:02 KST 2024
;; MSG SIZE  rcvd: 71
```
sorry, i am not sure i got them correctly because i am complete newbie and this is result you want i think.

**Reply by smaeda-ks** [reply]
if DNS_PROBE_FINISHED_NXDOMAIN was happening, I'd assume it was a typical DNS blocking. But even in the last one from @spoyodevelop, I don't see anything wrong.

@Socrad @unhyif do you know if they're also using Korea Telecom as well? And if so, could you get the same ones as well?

```
$ dig foo.vercel.app

$ dig foo.vercel.app @8.8.8.8
```

**Reply by Socrad** [reply]
I already changed domain name, and  ~.vercel.app domain works well and redirects to new domain now. But dnschecker.org still shows that some DNS don't work in Korea.

**Reply by smaeda-ks** [reply]
So far, no one has provided clear evidence of blocking, which is a problem. If DNS blocking is live, those two dig commands will show different outputs.

@Socrad @khj0426 Does the issue resolve if you switch to a public DNS service? 8.8.8.8/1.1.1.1 if so, it's most likely DNS filtering. But then we need the evidence. Please help us collect the logs so we can escalate it to the ISP.

**Comment by Socrad** [comment]
Today I checked connection via dnschecker.org, and it seems that some Korean DNS don't work.

**Comment by unhyif** [comment]
2 users of my service reported having trouble accessing the page on May 12th at 18:11 and May 13th at 09:30 (UTC+9).

**Comment by khj0426** [comment]
Today I checked my all vercel domain not work.. it seems that some DNS not works

**Reply by smaeda-ks** [reply]
@khj0426 What's your ISP? Could you please run these and report back to us?

```
$ curl https://ipinfo.io

$ dig vercel.com

$ dig foo.vercel.app

$ dig foo.vercel.app @8.8.8.8
```

**Reply by khj0426** [reply]
```terminal
 {mongdiwoo} 🎉   ~/Documents/codingProject/dreaming_ver2/dreaming   feature/51 ±  curl https://ipinfo.io                    
{
  "city": "Paripark",
  "region": "Seoul",
  "country": "KR",
  "loc": "37.4883,126.8255",
  "org": "AS3786 LG DACOM Corporation",
  "postal": "08359",
  "timezone": "Asia/Seoul",
  "readme": "https://ipinfo.io/missingauth"
}%                                                                                                                                                                  
 {mongdiwoo} 🎉   ~/Documents/codingProject/dreaming_ver2/dreaming   feature/51 ±  dig vercel.com

; <<>> DiG 9.10.6 <<>> vercel.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 62224
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;vercel.com.                    IN      A

;; ANSWER SECTION:
vercel.com.             12      IN      A       76.76.21.9
vercel.com.             12      IN      A       76.76.21.61

;; Query time: 47 msec
;; SERVER: 10.0.100.1#53(10.0.100.1)
;; WHEN: Mon May 13 16:18:10 KST 2024
;; MSG SIZE  rcvd: 71

 {mongdiwoo} 🎉   ~/Documents/codingProject/dreaming_ver2/dreaming   feature/51 ±  dig foo.vercel.app

; <<>> DiG 9.10.6 <<>> foo.vercel.app
;; global options: +cmd
;; connection timed out; no servers could be reached
 ✘ {mongdiwoo} 🎉   ~/Documents/codingProject/dreaming_ver2/dreaming   feature/51 ±  dig foo.vercel.app @8.8.8.8

; <<>> DiG 9.10.6 <<>> foo.vercel.app @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 58213
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;foo.vercel.app.                        IN      A

;; ANSWER SECTION:
foo.vercel.app.         600     IN      A       76.76.21.9
foo.vercel.app.         600     IN      A       76.76.21.22

;; Query time: 68 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Mon May 13 16:18:38 KST 2024
;; MSG SIZE  rcvd: 75
```

**Comment by hhkwon** [comment]
Now, It works on LG ISP. Probably solved.

**Comment by r-4bb1t** [comment]
In my case, all services deployed using Vercel are still inaccessible via the LG network.

[Update 2024-05-19] I found the occurrence of this in the domain `76.76.21.22`, `76.76.21.241`.

**Comment by Huinno-ParkJinHyun** [comment]
again.., it happend 

fyi : 2024 05 20 17 50 ~ 18 30  (utc +9 seoul)  vercel serverless ip was blocked by isp (kt & lgu+)

**Reply by hiddenest** [reply]
Could you run this command and share the result?
https://github.com/vercel/community/discussions/6807#discussioncomment-9394209

**Comment by code-xhyun** [comment]
I hate Korea ISPs

**Comment by kgneng2** [comment]
The ISP has started blocking again. It's LG U+.....

FUCK.....

**Comment by developerjhp** [comment]
https://github.com/developerjhp/vercel-dns-isp-block-watch-dog
KT, SKT, LGU+ DNS 서버를 통해 Vercel로 배포된 도메인의 접근 가능 여부를 모니터링하고, 문제가 발생하면 Slack으로 알림을 보내는 코드를 만들었습니다.
(아쉽게도 serverless ip가 막히는지는 체크할 수 없습니다.. )
(isp에 의해서 block된 경우 aws나 netlify 로 배포된 리소스에 도메인을 연결하시는 방법이 그나마 가장 나은 해결책인 것 같습니다.)

github action은 cron 스케쥴링이 지멋대로여서.. 로컬에서 돌리시는 걸 추천드립니다~

**Comment by wjaykim** [comment]
It seems LG U+'s cellular network's default DNS(main: `106.101.122.4`, sub: `117.111.30.132`) doesn't return ip for `*.vercel.app` for now.

**Comment by amyegan** [comment]
Thank you all for reporting this. As [others mentioned earlier](https://github.com/orgs/vercel/discussions/6807#discussioncomment-9396547), we need more information. 

If you cannot access `vercel.app` domains, please run the following commands and share your results.
https://gist.github.com/smaeda-ks/5997018d5fe5dce44d047299829288e1

This will give us more evidence to share with the ISP showing what the issue is.
